cmd/server/http/policy.go

```diff
@@ -30,12 +30,19 @@ func applyAutoReleasePolicy(payload *payload, policySvc *policyinternal.Service)
 			return
 		}
 
-		logger = logger.WithFields("service", req.Service, "req", req)
-		logger.Infof("http: policy: apply: service '%s' branch '%s' environment '%s': apply auto-release policy started", req.Service, req.Branch, req.Environment)
-		id, err := policySvc.ApplyAutoRelease(ctx, policyinternal.Actor{
+		actor := policyinternal.Actor{
 			Name:  req.CommitterName,
 			Email: req.CommitterEmail,
-		}, req.Service, req.Branch, req.Environment)
+		}
+		subject := r.Context().Value(AUTH_USER_KEY).(string)
```

Comment:
This might be a good candiate for a `UserKeyFromContect()` function to hide the nasty lookup/casting stuff?